components/charts/ChartFigure/index.js

```diff
@@ -0,0 +1,22 @@
+import PropTypes from "prop-types";
+import * as Styled from "./styles";
+
+const ChartFigure = ({ caption, children, legend }) => {
```

Comment:
the ChartFigure wraps charts into a semantic figure that describes the chart with a caption